applications/CableNetApplication/tests/cable_net_test_factory.py

```diff
@@ -0,0 +1,106 @@
+from __future__ import print_function, absolute_import, division
+import KratosMultiphysics as KM
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.kratos_utilities as kratos_utils
+
+
+from KratosMultiphysics.CoSimulationApplication.co_simulation_analysis import CoSimulationAnalysis
+from KratosMultiphysics.StructuralMechanicsApplication.structural_mechanics_analysis import StructuralMechanicsAnalysis
+
+
+import KratosMultiphysics.CableNetApplication
+import cable_net_test_case
+import os
+
+have_dem_fem_dependencies = kratos_utils.CheckIfApplicationsAvailable("DEMApplication", "StructuralMechanicsApplication", "MappingApplication", "ExternalSolversApplication")
```

Comment:
```suggestion
have_dem_fem_dependencies = kratos_utils.CheckIfApplicationsAvailable("DEMApplication", "StructuralMechanicsApplication", "MappingApplication", "ExternalSolversApplication", "CoSimulationApplication")
```

Also then you should do the import of the CoSimAnalysis later